assets/src/apps/authoring/components/Flowchart/rules/create-three-try-workflow.ts

```diff
@@ -123,9 +122,7 @@ export const generateMaxTryWorkflow = (
         incorrect.destinationId,
         false,
         40,
-        extraOptions.maxAttempt == '1' && incorrect.feedback != DEFAULT_INCORRECT_FEEDBACK
-          ? incorrect.feedback
-          : options.threeTimesFeedback,
+        options.threeTimesFeedback,
```

Comment:
Reverting back changes done to handle the special case of max attempt = 1. With the new comment added in the card, we need to show the max attempt feedback.